### ISSUE TITLE
Feature Request : AutoColor Gradient Parent track

### ISSUE THREAD

**Issue by iNX974** [Jeff0S/sws#711]
I love the Autocolor, but for me I'm not satisfy with the parameter gradient which is blocked by the only color defined in the color management.
If the autocolor track gradient is based on the parent's colors track to enjoy much more multiple gradient color

**Comment by michalk-k** [comment]
I would welcome this extension too

**Comment by barletta** [comment]
Yes please. Being able to have a gradient on the children tracks based of its parent would be great!
